source/tests/pt_expt/infer/test_deep_eval_pt_checkpoint.py

```
@@ -0,0 +1,790 @@
+    def test_select_head_matches_single_task_forward(self) -> None:
+        rng = np.random.default_rng(GLOBAL_SEED + 1)
+        natoms = 4
+        coords = rng.random((1, natoms, 3)) * 8.0
+        cells = np.eye(3).reshape(1, 9) * 10.0
+        atom_types = np.array([i % 2 for i in range(natoms)], dtype=np.int32)
+
+        for head, src in (("head_a", self.model_a), ("head_b", self.model_b)):
+            # Build a DeepPot wrapping this DeepEval for end-to-end eval.
+            dp = DeepPot(self.pt_path, head=head)
+            de = dp.deep_eval
+            e, f, v = dp.eval(coords, cells, atom_types, atomic=False)
```

Ruff (0.15.11)

[warning] Unpacked variable `v` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)